packages/longhorn/templates/crd-template/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 1.1.2
+version: 1.2.0
```

Comment:
The main chart version will be overridden to 1.1.2 if we don't have this change, so we still need to update the longhorn-crd chart version to 1.2.0.

I would suggest keeping this change. WDYT? @jiaqiluo

Reply:
You are right :+1:

Reply:
Then do we still need to set this field? Retaining this field to `1.1.2` for v1.2.0 Chart looks weird and may lead to confusion.

Reply:
True, but it seems something needed to fix in https://github.com/rancher/charts-build-scripts because currently, it will check the version field by using helm lib during the `make prepare`.

This should be possible to enhance in the future. Leave to Rancher team.